service/dap/server.go

```diff
@@ -326,6 +354,11 @@ func (s *Server) triggerServerStop() {
 // TODO(polina): allow new client connections for new debug sessions,
 // so the editor needs to launch delve only once?
 func (s *Server) Run() {
+	if s.listener == nil {
+		go s.serveDAPCodec()
```

Comment:
Good question. We shouldn't require the same user here - this reverse mode will be used to launch the debug target and debug server as a root (with `sudo`).
The host:port the server opens accepts only one client. There is a small window before the reverse dap client is up and connects. But what the attacker can do with that is limited - we will block RunInTerminal request forwarding in the proxy-mode  implementation.

Reply:
And if somebody uses this mode directly outside of RunInTerminal set-up?
In any case, a comment in the code why you are skipping this would be helpful.